repository

```diff
@@ -45,6 +45,12 @@ echo '```' >> README.md
 test -z "$GITHUB_TOKEN" && exit
 
 git init -b "$BRANCH"
+
+if [ "$GIT_LFS" = "true" ]; then
+  git lfs install
+  git lfs track "*.deb" "*.ddeb"
+fi
```

Comment:
```suggestion
test -n "$GIT_LFS" && git lfs track "*.deb" "*.ddeb"
```

There should be no need to `git lfs install` when using the Debian package. Also I would just enable it if the variable is set.

Reply:
Good catch on `git lfs install`. I've kept the `if` statement as it follows the convention of the other variables in the action and it's more explicit than just checking it's set.

Reply:
Done in 47173e6d8474238b509508674d832128095233cb

Reply:
> I've kept the `if` statement as it follows the convention of the other variables in the action and it's more explicit than just checking it's set.

Huh, can you show me the other places?

Reply:
@jspricke SQUASH_COMMIT, just two lines after the GIT_LFS block

Reply:
well, that one has a more complex logic, look at the other parts ;).

Reply:
@jspricke In what way? The logic is the same, it takes a variable from the action.yml file and then acts on it, just checking that GIT_LFS is set doesn't say anything about its content. If I set it to the value `false` in the action.yml file, I expect it to be false.

Reply:
> You would need a translation layer somewhere if you want that.

And yet, there's no "translation layer" for SQUASH_HISTORY, so I don't don't understand this inconsistency. Both should follow the same convention.

> P.S.: Please don't mark this as resolved as long as we disagree.

I haven't seen any reasoning why this specific variable must be treated differently, so I don't understand any of this argument.